content/handbook/security/corporate/end-user-services/supported-apps/zoom/zoom-recording.md

```
@@ -0,0 +1,118 @@
+title: Zoom 録画
+description: Zoom 録画 - セットアップと設定
+upstream_path: /handbook/security/corporate/end-user-services/supported-apps/zoom/zoom-recording/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## 目次
+
+- [Zoom での録画](#recording-in-zoom)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)